### ISSUE TITLE
accented characters broken; 'use utf8;' has no effect in PDF::Table

### ISSUE THREAD

**Issue by westmj** [PhilterPaper/PDF-Table#48]
Appears still broken.

[accents-03-no use-utf8.pdf](https://github.com/kamenov/PDF-Table/files/3692615/accents-03-no.use-utf8.pdf)
[accents-02-use-utf8.pdf](https://github.com/kamenov/PDF-Table/files/3692616/accents-02-use-utf8.pdf)
[accents-01-use-utf8.pdf](https://github.com/kamenov/PDF-Table/files/3692617/accents-01-use-utf8.pdf)

[accents-tables.pl.txt](https://github.com/kamenov/PDF-Table/files/3692618/accents-tables.pl.txt)

The following issues might be related.
Hide
Example with utf8/cyrillic

#45 opened on Jun 14, 2018 by suricactus • updated on Jun 14, 2018

0
Words containing umlaut characters are rendered wrong

#15 opened on Jan 7, 2015 by kamenov • updated on Oct 8, 2015

1
Documented way to create a PDF::Table object gives errors

#17 opened on May 29, 2015 by philgoetz • updated on Jan 1, 2016

5
Warns when a table spills over on to the next page?

#42 opened on Mar 14, 2018 by djzort • updated on Mar 14, 2018

0
Related Issues Beta  or give us feedback.

**Comment by PhilterPaper** [comment]
This appears to be trying to use UTF-8 with corefont(). That doesn't work (only single byte encodings allowed for corefont, psfont, and bdfont). If you want to use UTF-8 character encoding, you must use ttfont() (or cjkfont()?).

No idea about #42 -- it's fairly old, is it still an issue?

**Comment by PhilterPaper** [comment]
As this ticket primarily concerns use of UTF-8 with corefonts, and that is actually a PDF::API2 and PDF::Builder issue, I am closing this ticket. 

TL;DR corefont, psfont, and bdfont are single byte encodings _only_.
